ArduPlane/quadplane.cpp

```diff
@@ -39,14 +39,24 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     AP_SUBGROUPPTR(pos_control, "P", 17, QuadPlane, AC_PosControl),
 
     // @Param: VELZ_MAX
```

Comment:
It is safe to just change it to `VELZ_MAX_UP`, not sure if its better to or not tho

Reply:
wanted to allow people who have changed the value from defualt to still use it when copying params back into a new firmware load... or a new board....thought changing the param name would prevent that

Reply:
discuss at devcall

Reply:
If one is down, the other should be up, otherwise it's just confusing, IMO.

Reply:
could parameter conversion somehow be used to convert Q_VELZ_MAX, someone tries to set that from an old param file into UP and DN of that value, allowing old param files to work with the new code that changes VELX_MAX to MX_UP?

Reply:
Nope. We don't have any parameter change hooks to do something like that. If we did we'd already have done a lot more sanity checking on some params when people try to set them.